### PR TITLE
fix: deploy cmd for prod env

### PR DESCRIPTION
**Pull request by mihai-sysbio**

### CODE REVIEW

**Review by e0 — APPROVED** [review]
We should also consider updating the other commands and documentation on Docker version requirement etc but of course we can merge in this urgent change first.

File: proj.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ function logs {
 
 function deploy-stack {
   generate-data
-  docker-compose -f docker-compose.yml -f docker-compose-prod.yml --context $1 up -d --build
+  docker --context $1 compose -f docker-compose.yml -f docker-compose-local.yml logs -f nginx
```
Comment (nanjiangshu) [inline proj.sh:44]: The change of `docker-compose-prod.yml` to `docker-compose-local.yml`, is that just a temporary fix? How is the SSL certificate set up in this case?

Comment (e0) [reply proj.sh:44]: Oh I didn't realize that haha. It wasn't intentional and we should probably not merge this in yet..

Comment (mihai-sysbio) [reply proj.sh:44]: also we  might not need the `logs -f nginx` at the end either

Comment (e0) [inline proj.sh:44]:
```suggestion
  docker --context $1 compose -f docker-compose.yml -f docker-compose-prod.yml up -d --build --force-recreate
```
Fixes the script and also adds the `--force-recreate` flag. This ensures that we do not use cached images when deploying.

Comment (mihai-sysbio) [reply proj.sh]: This is great.

**Review by nanjiangshu — APPROVED** [review]

**Pull request merged**